tests/5.1/target/test_target_is_accessible.F90

```diff
@@ -0,0 +1,72 @@
+!===--- test_target_is_accessible.F90 -------------------------------===//
+!
+! OpenMP API Version 5.1 Nov 2020
+!
+! This test checks that the omp_target_is_accessible device routine.
+! In this test the output of the target_is_accessible call should return
+! true because the storage indicated by the first and second arguements
+! is accessible by the targeted device. This test is closely adapdted
+! from the 5.1 OpenMP example sheet.
+!
+!//===---------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+PROGRAM test_target_is_accessible
+  USE iso_fortran_env
+  USE, INTRINSIC :: iso_c_binding
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(check_device() .NE. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION check_device()
+    INTEGER :: errors, i, isSharedMemory, N
+    INTEGER, POINTER :: fptr(:)
+    TYPE (C_PTR) :: ptr
+    INTEGER (C_SIZE_T) :: buf_size
+    INTEGER (C_INT) :: dev, check_test
+
+    errors = 0
+    isSharedMemory = 0
+    check_test = 2
+    N = 100
+    dev = omp_get_default_device()
+
+    !Assumes that on shared-memory systems, no copy is done
+    !$omp target map(to: isSharedMemory)
+    isSharedMemory = 1
```

Comment:
What is the intent here ?

Reply:
If the target system supports unified shared memory, the host will see this change, and thus isSharedMemory = check_test (line 51) = 1. 
If the target system does not support unified shared memory, the host will see isSharedMemory = 0, and check_test = 0 too.
In the host fallback case, the host will see isSharedMemory = 1, and check_test = 1 too.

Reply:
I don't think `isSharedMemory` is necessary. Since the program is to test `omp_target_is_accessible` not shared memory.